MekHQ/docs/AtB Stuff/stratcon-faq.txt

```diff
@@ -0,0 +1,62 @@
+StratCon reference and FAQ
+
+How do I get this started?
+Go to the AtB tab in Campaign Options and check "Use StratCon" at the very bottom.
+
+
+How do I use the StratCon tab?
+It's a graphical representation of one or more theatres of operations for your current AtB contract. It will have meaningful data once you arrive on the contract planet and the contract begins.
+
+Hexes where you've conducted recon are displayed as light gray, hexes where you have not are dark gray. Pro tip: if you want to reveal hexes faster, use a lance with the "scout" disposition, that'll reveal all hexes around the deployment hex.
+
+Allied facilities are displayed as green squares.
+Hostile facilities are displayed as red squares.
+Your forces deployed to the track are displayed as cyan circles.
+Pending scenarios against hostile forces are concentric red squares.
+
+
+That's great, how do I win?
+Under Integrated and House command, you need to keep your VP count positive. Every time you win a scenario which you didn't initiate, that's +1 VP. Every time you lose a scenario you didn't initiate, that's -1 VP. 
+
+Under Liaison command, you need to keep your VP count positive AND you're responsible for completing the given number of strategic objectives. On defensive contracts, this may include keeping allied facilities intact and under allied control. On offensive contracts, it's usually capturing/destroying facilities or winning specific scenarios. VPs are only affected by scenarios in which a liaison participates.
+
+Under Independent command, you're only responsible for completing strategic objectives.
+
+
+This scenario looks pretty tough, how do I bring in more forces?
+Right click on it and select 'manage scenario'. Now you have a couple of options:
+
+If your primary lance leader has leadership, and you have "auxiliary units" available (ones that are not of the same type as the main unit type of the primary force), you can deploy as many auxiliaries as you have leadership points.
+
+If your primary lance is in a 'defend' role, you can deploy up to 2 * (x + 1) your lance leader's tactics skill in infantry, battle armor squads and minefields. Any "left-over" defensive points are automatically converted into standard minefields.
+
+Finally, you can bring in additional lances, with rules as follows:
+- If the lance is already on the same track as the scenario, you can just deploy it as is.
+- If you have a support point or a victory point to spare, you can deploy the lance as is.
+- If the lance is in a 'fight' role, you can deploy it. However, on a 8- on a 2d6, it will add an extra negative modifier to the scenario.
```

Comment:
Just personal taste, but can we call it 8 or less instead of 8-? I find it can be confusing.